src/parser/WASMParser.h

```diff
@@ -17,7 +17,13 @@
 #ifndef __WalrusWASMParser__
 #define __WalrusWASMParser__
 
+#if !defined(NDEBUG)
+#include "interpreter/ByteCode.h"
+#endif
+
+#ifndef __WalrusModule__
```

Comment:
Why did you add this macro?
`__WalrusModule__` is already defined inside of `runtime/Module.h`